src/streams/mod.rs

```diff
@@ -66,6 +66,12 @@ pub struct RtpPacket {
     /// For incoming packets it's the time we received the network packet.
     pub timestamp: Instant,
 
+    /// Timing information from the most recent Sender Report(SR). The first part of the tuple is
+    /// the NTP time, the second part is the RPT timestamp that corresponds to the same timestamp.
+    ///
+    /// If no Sender Report(SR) has been received or this packet is being sent by str0m this is [`None`].
+    pub last_sender_timing: Option<(MediaTime, u32)>,
```

Comment:
I think MediaTime is better suited for RTP time and not for NTP time

Reply:
I think it's a doc error.

Reply:
But having said that, I think we should surface it as a proper struct in `MediaData` instead of a tuple. I'm thinking we might zhuzh up `SenderInfo` and release that.

The one thing I want to change is to interpret that u32 to _something good_, like `Instant` or `MediaTime`.

Reply:
https://github.com/algesten/str0m/pull/333/files#diff-5fac4aafc6e7f613e62fad8a848b9dfd4b5e625f210f7835428533b28015fda1R356

Reply:
It's not a doc error, we do use a MediaTime for the NTP time in `SenderInfo`. I also thought it was a bit weird, should be converted to `SystemTime`imo.

Here's the parsin code https://github.com/algesten/str0m/blob/9feeb86cae7f59992799f5f00a5a77a42c7080d1/src/rtp/rtcp/sr.rs#L110-L113